hardware/arduino/avr/platform.txt

```diff
@@ -67,6 +67,10 @@ recipe.objcopy.eep.pattern="{compiler.path}{compiler.objcopy.cmd}" {compiler.obj
 ## Create hex
 recipe.objcopy.hex.pattern="{compiler.path}{compiler.elf2hex.cmd}" {compiler.elf2hex.flags} {compiler.elf2hex.extra_flags} "{build.path}/{build.project_name}.elf" "{build.path}/{build.project_name}.hex"
 
+## Save hex
+recipe.hex.tmp_file={build.project_name}.hex
```

Comment:
Could you rename `hex` here to `output`? and also the related methods `saveHex(..)` to `saveOutput(..)` etc.

I'm wondering that the correct key for this recipe is independent from the output format of the build, so it should really be:

```
recipe.output.tmp_file={build.project_name}.hex
recipe.output.save_file={build.project_name}.{build.variant}.hex
```